### PR TITLE
Pipelines working with granite

### PR DESCRIPTION
**Pull request by RHRolun**
- New templates
- Updated images
- Updated langchain
- Added missing image bucket env var

### CODE REVIEW

**Comment by RHRolun** [comment]
Closes #113 #114

File: lab-materials/03/06/llm_usage.py [hunk 1]
```diff
@@ -24,11 +22,11 @@ def infer_with_template(input_text, template):
         verbose=False,
     )
     
-    PROMPT = PromptTemplate.from_template(template)
+    prompt = PromptTemplate(input_variables=["car_claim"], template=template)
```
Comment (rcarrata) [inline lab-materials/03/06/llm_usage.py:25]: @RHRolun why we're selecting only car_claim? will be only targeting this input_variable?

File: lab-materials/05/05-05/run_process_claims_pipeline.py [hunk 2]
```diff
@@ -11,10 +11,15 @@
 
 from process_claims_pipeline import process_claims_pipeline
 
-COMPILE=False
+COMPILE=True
+
+metadata = {
+        "claim_ids": 0,
+        "detection_endpoint": "https://some-endpoint",
```
Comment (rcarrata) [inline lab-materials/05/05-05/run_process_claims_pipeline.py:18]: @RHRolun is this expected to be this url, or it's just a mockup and any variable will work?

**Review by rcarrata — APPROVED** [review]
lgtm

File: lab-materials/05/05-05/run_process_claims_pipeline.py (hunk 2 above)
Comment (RHRolun) [reply lab-materials/05/05-05/run_process_claims_pipeline.py:18]: Mockup here, the users will enter the url to their served model when running the pipeline (this becomes default parameters which suggest the user what to enter)

File: lab-materials/03/06/llm_usage.py (hunk 1 above)
Comment (RHRolun) [reply lab-materials/03/06/llm_usage.py:25]: Correct, the templates only have a single variable in them and it's "car_claim".

**Pull request merged**